en/publish/publish-bytedance.md

```diff
@@ -0,0 +1,122 @@
+# Publish to ByteDance Mini Games
+
+Cocos Creator starts to support publishing games to **ByteDance Mini Games** from v2.4.2.
+
+The runtime environment of ByteDance Mini Game is an extension of ByteDance Mini Program, which provides the encapsulation of WebGL interfaces on top of the Mini Program environment, making the rendering ability and performance substantially improved. However, since these interfaces are encapsulated by the ByteDance team through self-developed native implementations, they are not equivalent to the browser environment.
+
+As the engine developer, in order to simplify the workload of developers as much as possible, the main work we have done for developers includes:
+
+- The engine framework is adapted to the ByteDance API and is purely at the game logic level, so developers do not need any additional modifications
+- Cocos Creator editor provides a quick packaging process to publish directly as a ByteDance 
+- Automatically loads emote assets, cached assets and the versioning of cached assets
+
+For details about ByteDance  access, please refer to [ByteDance Mini Game Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/registration).
+
+## Preparation
+
+- Download [ByteDance Developer Tools](https://microapp.bytedance.com/docs/zh-CN/mini-game/develop/developer-instrument/developer-instrument-update-and-download) and install it.
+
+- Refer to the document of [ByteDance Mini Game Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/registration) to register, login and apply for mini games on ByteDance Developer Platform.
```

Comment:
Refer to the document of [ByteDance Mini Game Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/registration) to register, login and apply for mini games on ByteDance Developer Platform.
-->
Refer to the [ByteDance Mini-Game Access Guide](https://microapp.bytedance.com/docs/zh-CN/mini-game/introduction/plugin-reference/registration) to register, login and apply for mini-games on the ByteDance Developer Platform.